scripts/upload_test_results/upload_to_github.py

```diff
@@ -0,0 +1,156 @@
+        # Use created repo to store results
+        origin = Repo(os.path.join(repo_dir))
+        upstream = Repo(os.path.join(repo_dir))
+        origin.git.checkout('master')
+        logging.info('Pull updates from the upstream ...')
+        upstream.git.pull()
+        logging.info('Push updates to the origin ...')
+        origin.git.push(origin_name, 'master')
+
+    try:
+        # Create a new branch for daily master results
+        new_branch_name = branch_git_title + commit_name
+        origin.create_head(new_branch_name)
+        logging.info('Checkout to the new branch %s', new_branch_name)
+        origin.git.checkout(new_branch_name)
+    except Exception:
+        logging.info('Branch for the current commit already existing, skip step')
+
+    # create full git directory path where your results will be stored
+    git_dir_path = os.path.join(repo_dir, 'results', ver_folder_name)
+    logging.info('Destination directory path to store file in Git %s', git_dir_path)
+
+    if not os.path.exists(git_dir_path):
+        os.mkdir(git_dir_path)
+        logging.info('Path created %s', git_dir_path)
+    else:
+        logging.info('Path already existing')
+
+    return git_dir_path, new_branch_name, origin
+
+def add_push_to_github(git_dir_path, origin, commit_msg, title_msg):
+    # Add your test results to the repo, commit them and then push them to create PR on Github
+    for j in glob.glob("{}/*.xml".format(file_path)):
+        file_name = os.path.basename((j))
+        logging.info("File name %s", file_name)
+
+        # Return path to its initial status
+        source_file_path = argument.res_dir
+        source_file_path = os.path.join(source_file_path, file_name)
+        logging.info('Source directory file path %s', source_file_path)
+        git_file_path = os.path.join(git_dir_path, file_name)
+        logging.info('Destination directory path file to commit %s', git_file_path)
+        shutil.copy(source_file_path, git_file_path)
+        origin.index.add(git_file_path)
+    try:
+        logging.info('Commit changes ...')
+        author = Actor(author_name, author_email)
+        origin.git.commit('-m', ver_folder_name + ": " + company_name + " "+ title_msg + \
+        "\n\n" + commit_msg + \
+        "\n\nSigned-off-by: " + author_name + " " + author_email, author=author)
+        logging.info('Push commit to the origin ...')
+        origin.git.push('origin', new_branch_name)
+        logging.info('Success! Go to your GitHub and create PR to merge the new origin branch with the upstream')
+        origin.git.checkout('master')
+
+    except Exception:
+        logging.critical('ERROR: Nothing to commit or that branch already on Github')
+        origin.git.checkout('master')
+        sys.exit(1)
+
+if __name__ == '__main__':
+    # Step 1. Read setup variables from config.txt file
+    config = configparser.ConfigParser()
+    config.read_file(open(r'config.txt'))
```

Comment:
I think it's better to use absolute path. When I run this script outside of this directory, it will raise
```
FileNotFoundError: [Errno 2] No such file or directory: 'config.txt'
```
Would it be more convenient to pass in the contents of `config.txt` via parameters?

Reply:
It will be too much, if to pass all the contents of config.txt via parameters. Information that changes once when you setup the environment will be in config.txt, information which you have to change every run will be passed using arguments (parameters).

Reply:
I didn't expect that script will be run outside of that directory. What do you mean about using absolute path? If you will run the script outside of its initial directory, you should enter the current path of the config.txt file manually. Or I misleading your input.

Reply:
The error mentioned above occurs when I run this python script as follows.
```
pyhton ./test_results/scripts/upload_to_github.py ...
```
The location of `config.txt` is fixed to this python script, 
```
    config_file = os.path.join(os.path.dirname(__file__), 'config.txt')
    config.read_file(open(config_file))
```

Reply:
Now I understand. Yes, that case I didn't test. Thank you for your input. I will fix that.